IT_wang.mod

```diff
@@ -61,7 +61,7 @@ ASSIGNED {
 }
 
 BREAKPOINT {
-	SOLVE states METHOD adams
+	SOLVE states METHOD euler
```

Comment:
@nrnhines: as `adams` is no longer supported in neuron 9.0, could you see if using Euler (or cnexp) is appropriate for above two mod files?

Reply:
Wouldn't `derivimplicit` be better?

Reply:
I will look into this in more detail and, hopefully be able to briefly look at method effects on the results.  I expect `derivimplicit` will be what we settle on. But it may be so non-stiff that euler would be fine (after all, adams was working). (do we still allow runge?). My prejudice is that a high order method is usually not very helpful to overall accuracy since voltage/current values are treated as constant over the time step. If the equations involve time dependent variables that are not STATE of the that mod file or explicit in those equations, the accuracy order of the integration method is not very meaningful.   Note that Cvode and IDA solve all states (including voltage) as a single global bag of equations and the SOLVE METHOD is not used for the equations contributed by the mod file.  Generally, I consider Cvode/IDA, with very small tolerance,  the result standard to compare fixed step results to.

Reply:
By the way, IT2_huguenard.mod and IT2_huguenard.mod are both in the two models
9889 lytton97
12631 lytton97b

Reply:
I see that these mod files in nrn-modeldb-ci use
```
IT2_huguenard.mod
SOLVE castate METHOD runge

IT_wang.mod
SOLVE states METHOD runge
```

Reply:
The results for 12631 lytton97b appear visually the same  for adams (version 8.2.2) and runge ( 9.0.dev-1250).
I suggest switching to runge.

(they also appear visually the same when changed to derivimplicit (IT_wang.mod) and cnexp (IT2_huguenard.mod))